k8s/deployments/mariadb.yaml

```diff
@@ -1,14 +1,3 @@
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: maria-db-data
-spec:
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: 5Gi
----
 apiVersion: apps/v1
```

Comment:
Sorry, I hadn't noticed this in earlier review. But shouldn't this entire file be deleted? The whole point is that we don't need a MariaDB container in k8s anymore